.github/workflows/ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - name: Publish unit test results
         if: >
           always() &&
+          github.event.sender.login != 'dependabot[bot]' &&
           ( github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id )
```

Comment:
Not related to this PR, but I wonder why we have `|| github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id` here?
If this is a pull request from a fork, then we won't be able to publish the unit test results anyway, right?

Reply:
You are right, that was a last-minute copy-paste error from the next step. To be fixed in #71.